hibernate-core/src/test/java/org/hibernate/jpa/test/lock/StatementIsClosedAfterALockExceptionTest.java

```diff
@@ -67,6 +67,7 @@ public void releaseResources() {
 
 	@Test(timeout = 1000 * 30) //30 seconds
 	@TestForIssue(jiraKey = "HHH-11617")
+	@RequiresDialectFeature(DialectChecks.SupportNoWait.class)
```

Comment:
can this be changed with `@SkipForDialect( value = CockroachDB192Dialect.class`

Reply:
done